examples/experimental/ProductionMonitoring/StreamTable.md

```diff
@@ -0,0 +1,94 @@
+
+A Weave StreamTable object enables continuous streaming of data from an application or service to W&B. This is an extension of the standard wandb.Table object to handle monitoring use cases. Instead of uploading a complete, immutable Table object once, you can append data repeatedly to the same StreamTable object with `.log([your data rows])`.
+
+A StreamTable of data serves as the foundation to build production monitoring Boards in Weave, like this example to recognize MNIST digits hand-drawn in an [interactive Jupyter notebook](../ProductionMonitoring/ProductionMonitoringConceptualOverview.ipynb).
+
+![small_prodmon_board](../../../docs/assets/mnist_pm_draw_hover.png)
+ 
+## Create a StreamTable
+
+The only required argument to create a StreamTable is the name of the StreamTable object. 
+
+```python
+from weave.monitoring import StreamTable
+st = StreamTable("my_entity_name/my_project_name/my_table_name")
+```
+If an entity (W&B username or shared team name) is not provided, this will attempt to  default to the current logged in entity on the local machine if there is one (via ~/.netrc file).
+
+## Log data to a StreamTable
+
+Call `.log()` to add rows to a StreamTable:
+
+```python
+st.log({"one_column_name" : "value_a", "another_column_name" : 7})
+st.log([
+    {"one_column_name" : "value_b", "another_column_name" : 19},
+    {"one_column_name" : "value_c", "another_column_name" : 28},
+    {"one_column_name" : "value_d", "another_column_name" : 36}])
+```
+`.log()` accepts a single dictionary or a list of dictionaries, where each dictionary entry corresponds to one row of the table. In each dictionary, the keys are column names and the values are the corresponding cell values.
+
+## Visualize the StreamTable
+
+The first call to `.log()` will return a Weave Panel URL, where you can view, edit, and save the resulting StreamTable as a Weave Board, of the form:
+
+View data at : https://weave.wandb.ai/?exp=get%28%0A++++%22wandb-artifact%3A%2F%2F%2Fstacey%2Fmesa%2Fmy_stream_table%3Alatest%2Fobj%22%29%0A++.rows
+
+![prodmon_tiny_table](../../../docs/assets/small_stream_table.png)
+
+
+Subsequent log calls will silently append these rows to the StreamTable instance.
+
+In a notebook, the StreamTable variable on a line by itself will return a Weave Panel view of the StreamTable. The StreamTable will contain all the logged columns and their values, as well as a `timestamp` column indicating when the row was logged. By default, rows will be ordered by oldest first. You can modify a StreamTable Panel as you would a wandb.Table from the UI: sort, filter, group, etc.
+
+**Note: Column display order is currently non-deterministic** over repeated views of the Panel. If you would like to modify the order, open the StreamTable Panel in a new window as a Board and edit/save a Board from this seed panel. There are two options to achieve this:
```

Comment:
Do you mind removing this block?

Reply:
which part of the block? the part where I explain what the StreamTable Panel shows in the UI, the note on display order , or both? As I user, I think both of these points are very helpful/would like to know them and not discover them. And sure, can remove.

Reply:
@tssweeney I softened the language here but want to keep some UI description.

Reply:
my guess is around non-deterministic column ordering, omitting that and including a call to action to use a board feels fine. This is something we may want to fix asap rather than document.